.github/template-files/config.yml

```
@@ -0,0 +1,45 @@
+conda/infrastructure:
+  # [required] general files and workflows
+  - CODE_OF_CONDUCT.md
+  - .github/workflows/cla.yml
+  - .github/workflows/update.yml
```

@jezdez a recent idea I had was to move the primary copy of the short version `CODE_OF_CONDUCT.md` to https://github.com/conda/governance:

```suggestion
conda/governance:
  # [required] community files
  - src: CODE_OF_CONDUCT_SHORT.md
    dst: CODE_OF_CONDUCT.md

conda/infrastructure:
  # [required] general workflows
  - .github/workflows/cla.yml
  - .github/workflows/update.yml
```

thoughts?

---

Think we could also create a `.github` repo for the org and add some of this shared content there

ref: https://github.com/orgs/community/discussions/22144#discussioncomment-3235649

---

We already have https://github.com/conda/.github for the org `README.md`.

I've been very hesitant to do more with that repo since only some select files are shared with the downstream projects and we sync more than just those files. In some cases we even sync different files to different projects (e.g. not all projects have the same issue templates).

My other concern was the git archives, AFAIK these wont include any of the special files defined in the `.github` repo so the archives would be an incomplete artifact.

---

> @jezdez a recent idea I had was to move the primary copy of the short version CODE_OF_CONDUCT.md to [conda/governance](https://github.com/conda/governance):

Yeah, now that we have conda/governance, we should use it as the source of truth for governance docs like the CoC, it just didn't exist in the main org when we adopted the CoC.

I hear @jakirkham's point though about having a copy in .github though, it's not a loss to have a copy there as well, but for completeness’s sake, having a copy in every repo doesn't hurt.

---

ah I misunderstood

we already sync files to https://github.com/conda/.github using the old sync workflow (see the [workflow](https://github.com/conda/infrastructure/blob/main/.github/workflows/sync.yml) and the associated [config](https://github.com/conda/infrastructure/blob/main/.github/sync/config.yml))

all of the projects using the old sync workflow will be migrated to this new workflow soon

---

Hey all, this is a little messy, I disagree that we should add a short version of the CoC since that waters down what we mean when we referring to the Code of Conduct. The main CODE_OF_CONDUCT.md file should simply have a TLDR; section at the top of the file as I commented in https://github.com/conda/governance/pull/146#pullrequestreview-2207442086.